utils/hardcoded.ts

```diff
@@ -2,7 +2,7 @@ import { Address, Bytes } from "@graphprotocol/graph-ts";
 import { ProofOfHumanity as ProofOfHumanityContract } from "../generated/ProofOfHumanity/ProofOfHumanity";
 
 export const ProofOfHumanityAddress = Address.fromBytes(
-  Bytes.fromHexString("0x29defF3DbEf6f79ef20d3fe4f9CFa0547acCeC0D")
+  Bytes.fromHexString("0xB6412c84eC958cafcC80B688d6F473e399be488f")
```

Comment:
This wallet address is overwritten from the script using mustache (according to hardcoded.template.ts)